src/components/Comments/hooks/useCommentAction.js

```diff
@@ -40,14 +44,13 @@ const useCommentAction = () => {
         }
       }
     } catch (e) {
-      if (__DEV__) {
-        console.log(e, 'eman');
-      }
+      console.log(e, 'eman');
     }
   };
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The console log statement `console.log(e, 'eman');` should be wrapped in a condition to check if the environment is development. This prevents unnecessary logging in production which can lead to performance issues and exposure of potentially sensitive information.
```diff
-      console.log(e, 'eman');
+      if (__DEV__) {
+        console.log(e, 'eman');
+      }
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->